bindings/c/test/mako/zipf.h

```diff
@@ -3,13 +3,22 @@
  * https://github.com/brianfrankcooper/YCSB
  */
 
+#ifdef __cplusplus
+extern "C" {
+#endif
+
 #ifndef ZIPF_H
```

Comment:
IMHO, these "once" check should be the most outer condition.

Reply:
Fix. Thanks.